### ISSUE TITLE
Application env vars missing in configManagementPlugins

### ISSUE THREAD

**Issue by msiebeneicher** [argoproj/argo-cd#1950]
**Describe the bug**
Application env vars missing in configManagementPlugins. 
Also the setting will not displayed in the UI.

**To Reproduce**
I used a similar setup like here described: https://github.com/argoproj/argo-cd/blob/master/docs/user-guide/config-management-plugins.md

Test configuration plugin:

`kubectl edit configmaps argocd-cm`
```
configManagementPlugins: |
    - name: gateway-manifest-download
      init:
        command:  [ sh, -c ]
        args: [ "env > /tmp/env_command.txt" ]
      generate:
        command: [sh, -c]
        args: [ "env > /tmp/env_generate.txt" ]
```

Manually added env var settings to app by `kubectl edit applications.argoproj.io my-app`
```
Spec:
  Destination:
    Namespace:  ns
    Server:     https://foo.bar
  Project:      default
  Source:
    Path:  /
    Plugin:
      Env:
        Name:        FOO
        Value:        bar
      Name:           my-plugin
```

**Expected behavior**
Created files should contain the setted env vars from application.

**Screenshots**
![image](https://user-images.githubusercontent.com/5821183/61473035-e6d56500-a985-11e9-86c5-fa84f03d8c0f.png)

**Version**

```shell
argocd: v1.0.2+e0bd546.dirty
  BuildDate: 2019-06-14T17:13:49Z
  GitCommit: e0bd546a07818ec06a27c2b3033454e3eb1c4152
  GitTreeState: dirty
  GoVersion: go1.11.4
  Compiler: gc
  Platform: linux/amd64
```

**Comment by alexec** [comment]
This is only available in v1.2.

**Comment by msiebeneicher** [comment]
> This is only available in v1.2.

Ok - Is there any kind of workaround until the v1.2.0 will come out?

**Comment by alexec** [comment]
Your tool could read some file from the repo maybe? Or use the `ARGOCD_APP_NAMESPACE` to look up the variables?
